workers/data_refinery_workers/processors/test_smasher.py

```diff
@@ -610,6 +610,103 @@ def test_no_smash_dupe(self):
         for column in final_context['original_merged'].columns:
             self.assertTrue('_x' not in column)
 
+    @tag("smasher")
+    def test_no_smash_dupe_two(self):
```

Comment:
I'm just wondering, what's this testing that isn't already tested? I'm not quite sure what the test function's name means.

Reply:
The test reflects an actual situation encountered where the tiles are the same, it also uses a QN target and has qn_normalize `true` on the dset

Reply:
Ah okay. I don't think there's anything in the code which explains this. The code looks like it's operating on two different files. If you could add a comment or docstring that would be helpful.